fastcdc/fastcdc.h

```diff
@@ -29,8 +29,11 @@
 namespace cdc_ft {
 namespace fastcdc {
 
-static constexpr uint32_t default_mask_stages = 7;
-static constexpr uint32_t default_mask_lshift = 3;
+// These are recommended and widely used multipliers for LCGs from
+// http://citeseer.ist.psu.edu/132363.html. They replace the gear lookup
```

Comment:
This site doesn't seem to exist anymore.

Reply:
Sigh! It seems pretty hard to find stable links to academic papers that aren't locked behind paywalls. Perhaps This one is better?

https://www.ams.org/journals/mcom/1999-68-225/S0025-5718-99-00996-5/S0025-5718-99-00996-5.pdf

Reply:
I've updated this to the link that seems to work.